src/main/java/org/opensearch/knn/index/memory/NativeMemoryEntryContext.java

```diff
@@ -75,6 +91,18 @@ public static class IndexEntryContext extends NativeMemoryEntryContext<NativeMem
         @Getter
         private final String modelId;
 
+        @Setter
+        @Getter
```

Comment:
Could you remove @Setter here?
This variable should be modified from outside other than testing.
Instead of having a public scope getter, please find another way to change it in testings.